theme/docs/conf.py

```diff
@@ -63,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'0.0.0'
+version = u'2.9.0'
```

Comment:
I'm not clear on why `version` and `release` needs to be changed to `2.9.0`.

Also `2.9.0` seems as arbitrary as `0.0.0`.

Reply:
I'm also not clear on that but my understanding is that this file has no impact on the doc that we generate. It's only used for generating the doc used for live-previewing the theme.

@bame-da, do you know if keeping it to 0.0.0 is ok?